deploy/olm-catalog/codeready-toolchain-operator/0.0.1/codeready-toolchain-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -68,35 +68,6 @@ spec:
     spec:
       clusterPermissions:
       - rules:
-        - apiGroups:
```

Comment:
the `operators.coreos.com` resources are namespaced, so I guess they did not need to be part of the cluster permissions anyways.